include/triangle_based_object.h

```diff
@@ -0,0 +1,13 @@
+#ifndef TRIANGLE_BASED_OBJECT_H
+#define TRIANGLE_BASED_OBJECT_H
+
+#include "scene_object.h"
+#include "triangle.h"
+#include <vector>
+
+class TriangleBasedObject : SceneObject {
+  protected:
```

Comment:
Already fixed in later commit 😌  
![skarmavbild 2016-09-17 kl 02 05 47](https://cloud.githubusercontent.com/assets/7224052/18604300/4c618b1e-7c7b-11e6-8821-5bdd5fd5601c.png)